packages/babel-plugin-proposal-decorators/test/fixtures/legacy-class-constructors/options.json

```diff
@@ -1,5 +1,8 @@
 {
-  "presets": [["env", { "targets": { "browsers": "ie 6" } }]],
+  "presets": [["env", {
+    "targets": { "browsers": "ie 6" },
+    "exclude": ["proposal-class-properties", "proposal-private-methods"]
+  }]],
```

Comment:
Can we remove `env` or use `env` + `decorators`? We have to tough this test again when private-in is stage 4.

Reply:
I'll use assumptions :+1: